# the pypi package is misconfigured and lists cython and nose as run dependencies.

diff --git a/setup.py b/setup.py
index 96fc73c..466b54f 100644
--- a/setup.py
+++ b/setup.py
@@ -14,8 +14,9 @@ extensions = [Extension('hdmedians.geomedian',
 
 setup(name='hdmedians',
       packages=find_packages(),
-      setup_requires=['nose>=1.0', 'Cython>=0.23'],
-      install_requires=['numpy', 'Cython>=0.23'],
+      setup_requires=['Cython>=0.23'],
+      install_requires=['numpy'],
+      test_requires=['nose>=1.0'],
       version='0.14.2',
       description='High-dimensional medians',
       url='http://github.com/daleroberts/hdmedians',
